Firmware_Bootstrap_Loader/faradaybsl/createtiscripter.py

```diff
@@ -0,0 +1,115 @@
+#-------------------------------------------------------------------------------
+# Name:        module1
+# Purpose:
+#
+# Author:      Brent
+#
+# Created:     03/08/2016
+# Copyright:   (c) Brent 2016
+# Licence:     <your licence>
```

Comment:
No license? GPLv3 buddy! Not sure on copywrite. Likely either you're full name or FaradayRF?